appengine/memcached/app.standard.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app_yaml]
+runtime: nodejs8
+
+# [START env_variables]
+env_variables:
+  # If you are using the App Engine Memcache service,
+  # uncomment this section and comment out the other Memcache variables.
+  #USE_GAE_MEMCACHE: 1
+  #GAE_MEMCACHE_HOST: 35.190.255.1
+  #GAE_MEMCACHE_PORT: 11211
+  MEMCACHE_URL: your-memcache-url
+  # If you are using a Memcached server with SASL authentiation enabled,
+  # fill in these values with your username and password.
+  MEMCACHE_USERNAME: your-memcache-username
+  MEMCACHE_PASSWORD: your-memcache-password
```

Comment:
Ditto to "do not check into source control" comment

Reply:
Done :)